examples/on-conflict-clause/on_conflict_clause.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/io;
+
+type Student record {|
+    readonly int id;
+    string name;
+    int score;
+|};
+
+public function main() {
+    error onConflictError = error("Key Conflict", message = "record with same key exists.");
+
+    table<Student> students = table [
```

Comment:
Need to update this BBE after fixing https://github.com/ballerina-platform/ballerina-dev-website/issues/6026